pkg/storage/chunk/gcp/instrumentation.go

```diff
@@ -52,13 +53,24 @@ func bigtableInstrumentation() ([]grpc.UnaryClientInterceptor, []grpc.StreamClie
 
 func gcsInstrumentation(ctx context.Context, scope string, insecure bool, http2 bool) (*http.Client, error) {
 	// start with default transport
```

Comment:
Is this comment now stale?

Can we add a comment here why we need to do this? You spent a long time debugging this, and I worry that if someone reverts this later they might re-introduce the weird bug you spent ages debugging.